ssl/ssl_test.cc

```diff
@@ -6510,6 +6510,18 @@ TEST_P(SSLVersionTest, SSLPendingEx) {
   ASSERT_EQ(buf_len, (size_t)2);
   EXPECT_EQ(3, SSL_pending(client_.get()));
   EXPECT_EQ(1, SSL_has_pending(client_.get()));
+
+  // 0-sized IO with valied inputs should succeed but not read/write nor effect
+  // buffer state. However, NULL outlen pointer should fail.
```

Comment:
Nit:
```suggestion
  // 0-sized IO with valid inputs should succeed but not read/write nor effect
  // buffer state. However, NULL outlen pointer should fail.
```
Also I'm assuming `outlen` refers to `read_bytes/written` here?

Reply:
yes, i can and should be more precise.